java-util/pom.xml

```diff
@@ -85,6 +85,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.datasketches</groupId>
```

Comment:
In https://github.com/druid-io/druid/pull/3915 we came to agreement that `Memory` should be in it's small dedicated library, not datasketches